### PR TITLE
ENH: add atef config sync script

### PR DESCRIPTION
**Pull request by tangkong**
Adds atef config sync script, adds to cron tab

### CODE REVIEW

File: sync_atef_config.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+source /cds/group/pcds/engineering_tools/latest-released/scripts/pcds_conda
+source "${HOME}/.pcdshub.sh"
+
+bsd_config_directories=(
```
Comment (ZLLentz) [inline sync_atef_config.sh:6]: maybe some copy/paste relics here

Comment (tangkong) [reply sync_atef_config.sh:6]: I was paying attention I swear

Comment (ZLLentz) [reply sync_atef_config.sh:6]: everything else looks good

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
merging, pulling, adjusting my pcdshub pushes crontab

**Pull request merged**